plugins/inventory/falcon_discover.py

```diff
@@ -0,0 +1,378 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, CrowdStrike Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+name: falcon_discover
+short_description: CrowdStrike Falcon Discover inventory source
+description:
+  - Query asset details from the CrowdStrike Falcon Discover API. To learn more about Falcon Discover and
+    Exposure Management, see the
+    L(Falcon documentation,https://falcon.crowdstrike.com/documentation/page/f2197af5/asset-management-overview-discover)
```

Comment:
Should document public vs. private IP behavior here in the primary docs section, as well as skipping hosts with no IP address (for completeness).

Reply:
I put it down in the notes portion - makes more sense in terms of describing a current functionality as opposed to describing what the module does.